hyperspy/tests/signal/test_eds_tem.py

```diff
@@ -327,3 +327,48 @@ def test_manual_remove_element(self):
         assert (
             sorted(s._xray_markers.keys()) ==
             ['Zn_Ka', 'Zn_La'])
+
+
+@lazifyTestClass
```

Comment:
The decorator looks for every test (i.e. a method that starts with `test...`), and creates its copy (which will be named `test_lazy...`) and prepends code that transforms all signals in the class as lazy. 

So these tests do not work for lazy because of two things:
 1. No `setup_method`, hence no signals to "lazify" to begin with.
 2. The original code is not adapted for lazy signals in general. I thought about it for around an hour at some point, and from what I can tell it is __much__ more difficult (but possible) to make it work lazily efficiently.

So currently it just runs the tests normally twice.